Automattic/sensei
# Avoid running WordPress Dependencies Report on trunk

### Changes proposed in this Pull Request

* Remove dependencies report jobs from the `js.yml` workflow, so it doesn't run on `trunk`;
* Add the dependencies report jobs to the `dependencies-report.yml` workflow;

### Testing instructions

Make sure the jobs still run on this PR (note that it shouldn't comment on the PR, it should just run as a traditional check). It shouldn't run on `trunk` anymore, after this change.

### Notes

I assigned Nexus as a review because this is more related to the janitorial. More context here: p1675161760814229-slack-C07418EJ0
